js/index.js

```diff
@@ -7390,7 +7386,7 @@
             ReadVCR.fromObject = function fromObject(object) {
                 if (object instanceof $root.api.ReadVCR)
                     return object;
-                return new $root.api.ReadVCR();
+                return $root.api.ReadVCR.fromObject();
```

Comment:
Will cases like this cause infinite recursion?

Reply:
oh yes they will certainly cause infinite recursion. tightened the search to avoid those!